res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.recipe

```diff
@@ -0,0 +1,117 @@
+operand {
+  name: "Input"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 1
+    dim: 1440
```

Comment:
For faster execution in luci-interpreter, can we have smaller single digit value instead of `1440`?

Reply:
Okay, I'll change 1440 -> 4.